Add support for UR8LONG

urfeex opened the pull request:
Closing #318

urfeex committed:
Add support for UR8LONG

urrsk commented on urdf/ur_mocked.urdf.xacro:
```diff
@@ -6,7 +6,7 @@
   <xacro:include filename="$(find ur_description)/urdf/ros2_control_mock_hardware.xacro" />
 
-  <!-- possible 'ur_type' values: ur3, ur3e, ur5, ur5e, ur7e, ur10, ur10e, ur12e, ur16e, ur15, ur20, ur30 -->
+  <!-- possible 'ur_type' values: ur3, ur3e, ur5, ur5e, ur7e, ur8long, ur10, ur10e, ur12e, ur16e, ur20, ur30 -->
```
- [x] UR15 is missing
- [x] Either group them by generation or payload

urrsk commented on urdf/ur.urdf.xacro:
```diff
@@ -5,7 +5,7 @@
    <xacro:include filename="$(find ur_description)/urdf/ur_macro.xacro"/>
 
-   <!-- possible 'ur_type' values: ur3, ur3e, ur5, ur5e, ur7e, ur10, ur10e, ur12e, ur16e, ur15, ur20, ur30 -->
+   <!-- possible 'ur_type' values: ur3, ur3e, ur5, ur5e, ur7e, ur8long, ur10, ur10e, ur12e, ur16e, ur15, ur20, ur30 -->
```
- [x] Either group them by generation or payload

urfeex committed:
Reorder model listings by generation, payload

The list of supported robot models has been growing over time. This commit
sorts them by generation (CB3, e-Series and UR series) and each generation
by payload.
